bazel/repositories.bzl

```diff
@@ -402,6 +408,33 @@ cc_library(
         **location
     )
 
+    # Parser dependencies
+    http_archive(
+        name = "rules_antlr",
+        sha256 = "7249d1569293d9b239e23c65f6b4c81a07da921738bde0dfeb231ed98be40429",
+        strip_prefix = "rules_antlr-3cc2f9502a54ceb7b79b37383316b23c4da66f9a",
+        urls = ["https://github.com/marcohu/rules_antlr/archive/3cc2f9502a54ceb7b79b37383316b23c4da66f9a.tar.gz"],
```

Comment:
Use latest 0.5.0
```
        name = "rules_antlr",
        sha256 = "26e6a83c665cf6c1093b628b3a749071322f0f70305d12ede30909695ed85591",
        strip_prefix = "rules_antlr-0.5.0",
        urls = ["https://github.com/marcohu/rules_antlr/archive/0.5.0.tar.gz"],
```

Reply:
Upgrading this fails to compile because "headers" is no longer supported in 0.5.0.  CEL needs to be upgraded first to support 0.5.0.  Added TODO to upgrade this when CEL is upgraded.

Reply:
Google hasn't imported new ANTLR, so it might take some time to upgrade Google. Current runtime has been fuzzed extensively, so I'm not too worried about missing some bugs.

Reply:
I don't think this will break Google import, since we only allowlist certain extensions and Wasm is not yet on this. @jplevyak  we should follow-up online on what it takes to import.

Reply:
I mean cel-cpp is a direct export from google3, so it is linked against internal google3 patched ANTLR which is not based on 0.5.0. We can import 0.5.0 into google3 and then link cel-cpp against that runtime. I am not sure what it takes to do a security review of a parser runtime though. When I looked briefly into that, it looked like ANTLR was lenient with breaking changes in a minor version.

Reply:
Right; are we just talking about BUILD files and rules_antlr here? If so, being behind the last release version isn't a security worry.